src/components/[guild]/Requirements/components/GuildCheckout/components/FeesTable.tsx

```diff
@@ -48,7 +48,13 @@ const FeesTable = ({
       </Button>
 
       <Collapse in={isOpen} animateOpacity>
-        <TableContainer borderWidth={1} borderRadius="xl" mt={2} bgColor={bgColor}>
+        <TableContainer
+          borderWidth={1}
+          borderRadius="xl"
+          mt={2}
+          bgColor={bgColor}
+          whiteSpace={"normal"}
```

Comment:
Love this solution, but now the "multiplier" line looks strange (the "x" and the number after it should be in one line)
<img width="452" alt="image" src="https://github.com/guildxyz/guild.xyz/assets/92519134/4b08caef-8765-4cd7-920e-41c7d31bdbe4">

Reply:
Actually I'm not sure if we should change the `FeesTable` component or create a new one specifically for dynamic rewards. I can also try to make only the first column break word (on dynamic rewards only). What do you think @BrickheadJohnny ?

Reply:
I think we can keep using the `FeesTable` component (maybe we could find a more general name for it, but we can leave it as is for now), just make the 1st column break word 👍